tests/pytests/test_aggregate.py

```diff
@@ -979,32 +979,40 @@ def testResultCounter(env):
     #env.expect('FT.AGGREGATE', 'idx', '*', 'FILTER', '@t1 == "foo"').equal([0])
 
 def test_aggregate_timeout():
-    if VALGRIND:
-        # You don't want to run this under valgrind, it will take forever
-        raise unittest.SkipTest("Skipping timeout test under valgrind")
+    def test_timeout_env(env):
```

Comment:
RLTest is looking for functions with a prefix of `test`, consider renaming (although it should be fine since it is a nested function).